src/analysisd/analysisd.c

```diff
@@ -97,6 +158,19 @@ static unsigned int hourly_events;
 static unsigned int hourly_syscheck;
 static unsigned int hourly_firewall;
 
+/* EPS limits struct */
+typedef struct _limits_t {
```

Comment:
This structure definition can be moved to the .h file.

Reply:
Done